code/modules/clothing/spacesuits/chronosuit.dm

```diff
@@ -171,15 +164,6 @@
 	else
 		finish_chronowalk(user, to_turf)
 
-
-/obj/item/clothing/suit/space/chronos/proc/create_phase_underlay(var/mob/user)
```

Comment:
What was going on here?

Reply:
😠 this ruined the animation for no reason
it was a blank white silhouette of you to go underneath you when teleporting so that you are completely white when you come in and slowly turn teal while gaining details, then you quickly fade to normal colors, and mrp ruined it!!! 💢